src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -728,6 +728,38 @@ object Desugar {
               }
             } yield in.Seqn(Vector(dPre, exprAss, clauseBody))(src)
 
+          case PGoStmt(exp) =>
+            def unexpectedExprError(exp: PExpression) = violation(s"unexpected expression $exp in go statement")
+
+            def getArgs(args: Vector[PExpression]): Writer[Vector[in.Expr]] = {
+              sequence(args map exprD(ctx)).map {
+                // go function chaining feature
+                case Vector(in.Tuple(targs)) if args.size > 1 => targs
```

Comment:
This line has a small error, there is no point in comparing `args.size` to 1. Instead, I wanted to compare the number of parameters in the function definition to 1